app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -140,6 +143,20 @@ public void updateFromItem(final InfoItem infoItem,
             itemPublishedTime.setText(item.getTextualUploadDate());
         }
 
+        if (item.getReplies() != null) {
+            itemContentReplyButton.setVisibility(View.VISIBLE);
+            itemContentReplyButton.setOnClickListener(
+                    view -> itemBuilder.getOnCommentsReplyListener().selected(item)
+            );
+            final int replyCount = item.getReplyCount();
```

Comment:
Shouldn't you `setVisibility(View.GONE)` when the reply count is 0?

Reply:
If no reply, `item.getReplies()` will return null, so not need to set at here.